template/page-community.html.jinja

```diff
@@ -407,3 +249,123 @@
     })
     </script>
 {% endblock analytics %}
+
+{% block endbody %}
+<script type="application/javascript" src="{{currentpage.prefix}}assets/js/contribute-carousel.js"></script>
+<script type="text/javascript">
+document.addEventListener("DOMContentLoaded", function() {
+const events = [
+  { "name": _("New Horizon: Innovate Without Limits: New Horizons Await"),      
```

Comment:
The `_(...)` function for translation doesn't work in this context. You can use it in `{% set ... %}` blocks which are handled by the Jinja preprocessor (which performs translation at build time) but you can't just use it in client-side JavaScript. The page is throwing an error in the browser console about `_` being undefined because of this.

One workaround is to wrap the entire events array here in `{{ ... }}` which evaluates it as Jinja syntax first, which will do translation and output a dictionary which happens to be compatible with JS syntax… as long as you consistently quote the keys (i.e. the unquoted key literals on lines 291–324 throw an error).

Another problem confounding this is that you have the event details defined in two places and this one is (currently) not being used, the one in `contribute-carousel.js` is. It's possible to work around that and make it use the translatable one from this file. But you have to do all of the following:

- remove the `events` definition from `contribute-carousel.js`
- use `var` instead of `const`
- move the script tag for `contribute-carousel.js` _below_ the one that defines `events`
- plus fix the things with translatability I mentioned in the previous paragraph.

Since that's complicated, I've pushed a commit ( 1e6c237e409b7755670e9f55034b42f3dad6e0c3 ) that does all of that. But you probably want to look at the updated events list and confirm that it works.